### PR TITLE
SPEC update: index table and protocol test coverage table

### PR DESCRIPTION
**Pull request by Rqnsom**
- change test index to table format
- add network protocol test coverage in the spec doc

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -67,7 +67,35 @@ The fuzz tests aim to buttress the message conformance tests with extra verifica
...
+| `<- B` | Algod sends a message `B` to Ziggurat's synthetic node                 |
+| `>> C` | Ziggurat's synthetic node broadcasts a message `C` to all its peers    |
+| `<< D` | Algod broadcasts a message `D` to all its peers                        |
+| `<>`   | Signifies a completed handshake, in either direction                   |
+
+## Network protocol test coverage
+
+|  Message           | Type              | Coverage | Tests                             |
+|--------------------|-------------------|----------|-----------------------------------|
+| Handshake          | HTTP              | ✅       | `C001`, `C002`, `c003`, `C004`    |
```
Comment (ljedrz) [inline SPEC.md:82]:
```suggestion
| Handshake          | HTTP              | ✅       | `C001`, `C002`, `C003`, `C004`    |
```

**Review by ljedrz — APPROVED** [review]

File: SPEC.md (hunk 1 above)
Comment (Rqnsom) [reply SPEC.md:82]: good catch!

Comment (Rqnsom) [reply SPEC.md]: done

File: SPEC.md [hunk 2]
```diff
@@ -67,7 +67,35 @@ The fuzz tests aim to buttress the message conformance tests with extra verifica
...
+| `>> C` | Ziggurat's synthetic node broadcasts a message `C` to all its peers    |
+| `<< D` | Algod broadcasts a message `D` to all its peers                        |
+| `<>`   | Signifies a completed handshake, in either direction                   |
+
+## Network protocol test coverage
+
+|  Message           | Type              | Coverage | Tests                             |
+|--------------------|-------------------|----------|-----------------------------------|
+| Handshake          | HTTP              | ✅       | `C001`, `C002`, `C003`, `C004`    |
+| UnknownMsgTag      | WS data (Tag: ??) | ❔       |                                   |
```
[the remark this answers is not in the document]
Comment (Rqnsom) [reply SPEC.md]: Thanks, I removed it from the table

**Review by ljedrz — APPROVED** [review]

**Pull request merged**